Portfolio/index.html

```diff
@@ -1,209 +1,236 @@
+                    <p><b>Integrated Firebase for user authentication and firestore forproduct data storage.</b></p>
+                    <p><b>Implemented secure payment processing with stripe API. </b></p>
 
+                    <div class="explore-btn">
+                        <button>Explore</button>
+                    </div>
+                </div>
 
+                <div class="project3">
 
                     <img src="/screen3.png" alt="css img" width="400px" height="400px">
-                    <h1><b>FinanceSage, Personal Finance Tracker </b></h1>
-                    <p>Svelte.js, Firebase</p>
-                    <p><b>Created a lightweight personal finance tracking web app using Svelte.js and Firebase.</b></p>
-                    <p><b>Users can log income and expenses, visualize financial data and set budget goals Firebase
-                            Authentication for user management and Firestore for data storage.</b></p>
+                    <div class="project-text3">
+                        <h1><b>FinanceSage, Personal Finance Tracker </b></h1>
+                        <p>Svelte.js, Firebase</p>
+                        <p><b>Created a lightweight personal finance tracking web app using Svelte.js and Firebase.</b>
+                        </p>
+                        <p><b>Users can log income and expenses, visualize financial data and set budget goals Firebase
+                                Authentication for user management and Firestore for data storage.</b></p>
 
-                    </b></p>
+                        </b></p>
 
-                    <div class="explore-btn">
-                        <button>Explore</button>
-                    </div>
+                        <div class="explore-btn">
+                            <button>Explore</button>
+                        </div>
 
 
+                    </div>
                 </div>
 
+                <div class="project4">
 
+                    <img src="/screen3.png" alt="css img" width="400px" height="400px">
+                    <div class="project-text4">
+                        <h1><b>FinanceSage, Personal Finance Tracker </b></h1>
+                        <p>Svelte.js, Firebase</p>
+                        <p><b>Created a lightweight personal finance tracking web app using Svelte.js and Firebase.</b>
+                        </p>
+                        <p><b>Users can log income and expenses, visualize financial data and set budget goals Firebase
+                                Authentication for user management and Firestore for data storage.</b></p>
+
+                        </b></p>
+
+                        <div class="explore-btn">
+                            <button>Explore</button>
+                        </div>
+                    </div>
+                </div>
     </section>
 
-
     <section class="contact" id="contact">
-        <h1>Get In Touch</h1>

```

Comment:
CSS properties in the .project2 class are placed in random order; consider organizing them for better readability.